loveship/components/dev/MakeMeItem.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { DropdownMenuButton } from '../overlays';
+import { DropdownMenuButton } from '../../components';
```

Comment:
such import will cause cycle dependencies, just make direct export from @epam/uui

Reply:
done

Reply:
Refactored DropdownMenu props: 
interface IDropdownMenuBody extends DropdownBodyProps, IHasCX, IHasChildren, IHasRawProps<HTMLAttributes<HTMLDivElement>>, IHasForwardedRef<HTMLDivElement> {
    minWidth?: number;
    closeOnKey?: React.KeyboardEvent<HTMLElement>['key'];
}
and now we can setup minWidth for container like that:  <DropdownMenuBody minWidth={ 90 } { ...dropdownProps }>